keyboards/primekb/prime_r/readme.md

```diff
@@ -12,7 +12,7 @@ Hardware Availability: [Prime_KB](https://www.primekb.com/)
 
 Make example for this keyboard (after setting up your build environment):
 
-    make prime_r:default
+    make primekb/prime_r:default
 
 See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
```

Comment:
Can you update this line to the following?

`See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).`

Reply:
The code in the folder isn't actually maintained by me, but I guess it's not a bad thing to add in the link to the guide.  Updated.